test/test_gillespie.jl

```diff
@@ -48,28 +51,3 @@ nums = Int[]
   push!(nums,sol[end][3])
 end
 println("Direct Jumps: $(mean(nums))")
-
-
-using Gillespie
```

Comment:
This is for testing against Gillespie.jl. Please leave it in.

Reply:
Ok, figured it had something to do with the old GuillespieProblems.